src/utils/getMetaAggregatorRoutes/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { Currency, CurrencyAmount, Price, TokenAmount } from '@kyberswap/ks-sdk-core'
+import JSBI from 'jsbi'
+
+import { RouteSummary } from 'types/metaAggregator'
+
+import { RawRouteSummary, RouteData } from './types'
+
+const toCurrencyAmount = function (currency: Currency, value: string | number): CurrencyAmount<Currency> {
```

Comment:
duplicate function 
search all `return TokenAmount.fromRawAmount(currency, JSBI.BigInt(value))`
duplicate in 3 places